src/Stryker.Core/Stryker.Core.UnitTest/Reporters/ConsoleReportReporterTests.cs

```diff
@@ -26,7 +26,7 @@ public void ConsoleReportReporter_ShouldPrintOnReportDone()
             chalkMock.Setup(x => x.DarkGray(It.IsAny<string>())).Callback((string text) => { output += text; });
             chalkMock.Setup(x => x.Default(It.IsAny<string>())).Callback((string text) => { output += text; });
 
-            var target = new ConsoleReportReporter(new StrykerOptions("", "ReportOnly", "", 1000, null, "debug", false, 1, 80, 60, 0), chalkMock.Object);
+            var target = new ConsoleReportReporter(new StrykerOptions(), chalkMock.Object);
```

Comment:
This would now get a different loglevel, not sure if it matters though?

Reply:
Test passes. If this does mater than I think original test needs to be changed, because it failed mutation ;)

Reply:
Hehe, check #104 for that. I'm not sure if it matters, maybe @richardwerkman cares to chime in?

Reply:
This discussion is one of the reasons I created this issue. All these values in the test make it unclear what is being tested. 

In this case the different loglevel has nothing to do with this UnitTest. All this test is about is the output of the reporter, which doesn't use any values from the options. So the default values are ok.